lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,20 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
+
 def binary_to_decimal(binary_array)
+  sum_of_digits = 0
+  power_of_digit = 7
+  binary_array.each do |digit|
+  sum_of_digits += digit * (2**power_of_digit)
+  power_of_digit -= 1
+  end
```

Comment:
This is good, just fixing indentation. 

```suggestion
  binary_array.each do |digit|
    sum_of_digits += digit * (2**power_of_digit)
    power_of_digit -= 1
  end
```